### PR TITLE
add second (and legacy) kafka consumer if env vars for it differ

### PR DESCRIPTION
**Pull request by gedge**
### What

(currently feature-flagged off in secrets)
add a second kafka consumer where secrets determine a second (differing) kafka server address
add TLS to kafka consumer, if requested

### How to review

test with features off

### Who can review

!gedge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,16 @@ Scripts for updating and debugging Kafka can be found [here](https://github.com/
 | Environment variable      | Default                              | Description
 | ------------------------- | -------------------------------------| ------------------------------
 | BIND_ADDR                 | :22200                               | The port to bind the application healhcheck endpoint to
-| KAFKA_ADDR                | http://localhost:9092                | The address of the kafka Instance
+| KAFKA_LEGACY_ADDR         | `localhost:9092`                     | The addresses of the kafka brokers (CSV) - non-TLS
+| KAFKA_LEGACY_VERSION      | `1.0.2`                              | The version of Kafka - non-TLS
+| KAFKA_ADDR                | `localhost:9092`                     | The addresses of the kafka brokers (CSV)
+| KAFKA_VERSION             | `1.0.2`                              | The version of Kafka
+| KAFKA_SEC_PROTO           | _unset_                              | if set to `TLS`, kafka connections will use TLS [1]
```
Comment (nshumoogum) [inline README.md:23]: Maybe link `[1]` to notes bullet point 1 - not necessarily obvious why this is here unless you review PR or look at code changes

**Review by nshumoogum — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gedge) [reply README.md]: is this any better?

Comment (justinpjose) [reply README.md]: You can use this as a reference (https://github.com/ONSdigital/dp-dimension-search-api/pull/88). This was reviewed by Nathan as well and approved

Comment (nshumoogum) [reply README.md]: Thanks Justin - yes I was thinking of the same changes made here

Comment (gedge) [reply README.md]: my markdown linter complains about inline-HTML for your method, so I prefer my method 😛 
(frankly, I think if people don't understand the original footnote syntax, then 🤷 but here we are...)

**Review by nshumoogum — APPROVED** [review]

**Pull request merged**